ideascube/serveradmin/catalog.py

```diff
@@ -253,6 +261,8 @@ def install(self, download_path, install_dir):
 
         for path in glob(os.path.join(datadir, '*', '{}*'.format(zimname))):
             new = path.replace(zimname, '{0.id}.zim'.format(self))
+            if Path(new).exists():
+                rm(new)
             os.rename(path, new)
```

Comment:
Weird.
Try to run tests, `test_kiwix_does_not_fail_if_files_already_exist` is failling with the error I paster above. I'll investigate a bit more.